spec/models/resource_spec.rb

```diff
@@ -60,13 +60,13 @@
   end
 
   describe ':attached_images' do
-    before { @resource.update_attributes(width: Setting['style.images_min_width'].to_i + 1) }
+    before { @resource.update_attributes(width: ENV['images_min_width'].to_i + 1) }
     it 'contains resources larger than half the standard width' do
       expect(Resource.attached_images.last).to eq(@resource)
     end
 
     context 'does not contain resources smaller than half the standard width' do
-      before { @resource.update_attributes(width: Setting['style.images_min_width'].to_i - 1) }
+      before { @resource.update_attributes(width: ENV['images_min_width'].to_i - 1) }
```

Comment:
Line is too long. [85/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.